src/applications/mhv-landing-page/components/HeaderLayout.jsx

```diff
@@ -67,7 +67,7 @@ const HeaderLayout = ({
               </div>
             </div>
           </div>
-          <div className="va-introtext">
```

Comment:
I didn't expect to see changes to the intro text based on the ticket. Couldn't we just move the `<va-alert-expandable />` outside of the va-introtext div?

Reply:
yes, will do